packages/web/src/pages/sign-in-page/SignInPageDesktop.tsx

```diff
@@ -22,28 +33,54 @@ const messages = {
   forgotPassword: 'Forgot password?'
 }
 
-export const SignInPageDesktop = () => {
+type SignInPageProps = {
+  title: string
+  description: string
+  canonicalUrl: string
+}
+
+export const SignInPageDesktop = (props: SignInPageProps) => {
+  const { title, description, canonicalUrl } = props
+  const { spacing } = useTheme()
+
   return (
-    <Flex h='100%' alignItems='center' justifyContent='center'>
+    <Page
+      title={title}
+      description={description}
+      canonicalUrl={canonicalUrl}
+      css={{
+        display: 'flex',
+        alignItems: 'center',
+        justifyContent: 'center'
+      }}
+    >
+      <BackgroundWaves />
+      <IconCloseAlt
+        color='staticWhite'
+        css={{
+          position: 'absolute',
+          left: spacing['2xl'],
+          top: spacing['2xl'],
+          zIndex: 1
+        }}
```

Comment:
curious, do you have a preference between this method (`useTheme` hook) vs this one?
I think they're functionally the same? So I guess its just ergonomics
```
        css={theme => {
          position: 'absolute',
          left: theme.spacing['2xl'],
          top: theme.spacing['2xl'],
          zIndex: 1
        }}
```

Reply:
yeah its mostly ergonomics so the css prop is not too complex looking... i personally reall like what material-ui does which is that user can do 
```
css={{
  left: '2xl'
}}
```
and they have built on top of the css prop to intercept that and use the theme variable... would be extra nice one day